backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -359,11 +366,15 @@ export class UserResolver {
       // TODO: this has duplicate code with sendResetPasswordEmail
       const emailOptIn = await createEmailOptIn(dbUser.id, queryRunner)
 
-      const activationLink = CONFIG.EMAIL_LINK_VERIFICATION.replace(
+      let activationLink = CONFIG.EMAIL_LINK_VERIFICATION.replace(
         /{code}/g,
         emailOptIn.verificationCode.toString(),
       )
 
+      if (redeemCode !== '') {
+        activationLink += '/' + redeemCode
+      }
```

Comment:
can we use `CONFIG.EMAIL_LINK_VERIFICATION.replace` here aswell?

Reply:
i have changed:

`  if (redeemCode) {
        activationLink += `/${redeemCode}`
      }`

Reply:
```
EMAIL_LINK_VERIFICATION=http://localhost/checkEmail/{optin}{code}
```
and
```
      const activationLink = CONFIG.EMAIL_LINK_VERIFICATION.replace(
        /{optin}/g,
        emailOptIn.verificationCode.toString(),
      ).replace(
        /{code}/g,
        redeemCode ? '/' + redeemCode : ''
      )
```

Reply:
?